### PR TITLE
Add robot-simulator exercise

### PR DESCRIPTION
**Pull request by BNAndras**
For #48in24

### CODE REVIEW

**Comment by BNAndras** [comment]
@ErikSchierboom, for your review. This is my first Groovy exercise port so let me know if I need to clean anything up.

File: exercises/practice/robot-simulator/src/test/groovy/RobotSimulatorSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        results.properties == expected.properties
+
+        where:
+        label                               || x    || y   || direction
+        "at origin facing north"            || 0    || 0   || "north"
+        "at negative position facing south" || -1   || -1   || "south"
+    }
+
+
+    @Unroll
```
Comment (ErikSchierboom) [inline exercises/practice/robot-simulator/src/test/groovy/RobotSimulatorSpec.groovy:17]: This also applies to the subsequent tests.

```suggestion
    @Ignore
    @Unroll
```

**Review by ErikSchierboom — COMMENTED** [review]
Just one small nit

File: exercises/practice/robot-simulator/src/test/groovy/RobotSimulatorSpec.groovy (hunk 1 above)
Comment (BNAndras) [reply exercises/practice/robot-simulator/src/test/groovy/RobotSimulatorSpec.groovy:17]: Thanks. I wanted to add it but VSCode seemed to indicate the ignore was invalid. I’ll take another look.

Comment (ErikSchierboom) [reply exercises/practice/robot-simulator/src/test/groovy/RobotSimulatorSpec.groovy:17]: The other exercises have it, so 🤷

Comment (BNAndras) [reply exercises/practice/robot-simulator/src/test/groovy/RobotSimulatorSpec.groovy:19]: Should be fixed now.

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**